### DISCUSSION TITLE
Changement d'URL canonique

### DISCUSSION THREAD

**Discussion by nriss** [Interop-Sante/hl7.fhir.fr.core#35] [Ideas]
Bonjour à tous,

Yohann a prévu un changement d'url canonique pour être davantage FHIR compliant avec [canonical]/[id]

Deux questions se posent :
- Est-ce painful à modifier ?
- Vaudrait-il pas mieux attendre d'avoir le serveur hl7.fr avec l'IG pour changer toutes les urls canoniques avec qqc de plus stable ?

**Comment by ypoiron** [comment]
De notre côté, l'impact est moindre de changer les URL
